app/src/main/java/org/cru/godtools/fragment/ToolsFragment.java

```diff
@@ -117,19 +138,30 @@ public void onCreate(@Nullable final Bundle savedInstanceState) {
             mMode = args.getInt(EXTRA_MODE, mMode);
         }
 
+        mSettings = Settings.getInstance(requireContext());
+
         startLoaders();
     }
 
     @Override
     public View onCreateView(@NonNull final LayoutInflater inflater, @NonNull final ViewGroup container,
                              @Nullable final Bundle savedInstanceState) {
-        return inflater.inflate(R.layout.fragment_tools, container, false);
+        mToolsBinding = FragmentToolsBinding.inflate(inflater, container, false);
+        return mToolsBinding.getRoot();
```

Comment:
now that you are using data binding you need to clean up and unset the data binding in `onViewDestroyed()` to prevent it from leaking views